src/client.js

```diff
@@ -3,11 +3,22 @@ import React from 'react';
 import { hydrate } from 'react-dom';
 import { ClientApp } from './app/containers/App';
 import routes from './app/routes';
+import {
+  reloadAfterInactivity,
+  resetInactivityTimer,
+} from './invalidationStrategy';
 
 const data = window.SIMORGH_DATA || {};
 const root = document.getElementById('root');
 
-hydrate(<ClientApp data={data} routes={routes} />, root);
+hydrate(
+  <ClientApp data={data} routes={routes} onNewRoute={resetInactivityTimer} />,
+  root,
+);
+
+reloadAfterInactivity({
```

Comment:
Im sure hydrate is async, so this shouldnt matter, but we should put `reloadAfterInactivity` before it

Reply:
Oh, in case it errors? I was going for a "if it takes ages to hydrate the client, we don't want to have started the timer too early - doesn't seem fair on the user" - but if the timer is an hour then the difference is negligible 😅 

I guess putting it first _should_ mean the reload is more likely to happen in the case of an error happening in the hydration...